encrypted-config-value/src/main/java/com/palantir/config/crypto/EncryptedValue.java

```diff
@@ -54,10 +56,10 @@ public static EncryptedConfigValue fromString(String encryptedValue) {
      * @param encryptedString - an encrypted string to wrap.
      * @return an EncryptedConfigValue for this encrypted string
      */
-    public static EncryptedConfigValue fromEncryptedString(String encryptedString) {
-        EncryptedConfigValues.getBytesAndValidate(encryptedString); // validate is base-64
+    public static EncryptedValue fromEncryptedString(String encryptedString) {
+        Base64Utils.checkIsBase64(encryptedString);
```

Comment:
so we should actually do this check using an `@Immutables.Check` function so that it's run when you set `encryptedValue`.